### ISSUE TITLE
Filters are not accessible

### ISSUE THREAD

**Issue by thibaultamartin** [matrix-org/matrix.org#1788]
https://github.com/matrix-org/matrix.org/assets/1688309/b7caadb7-f302-4f55-93f5-0375d5781764

- [ ] Turn the filters into `button`s
- [ ] Enter toggles the open/closed state of the menu
- [ ] Esc closes the menu
